jest.config.ts

```diff
@@ -1,31 +1,13 @@
-import { Config } from "@jest/types"
+import { Config } from "@jest/types";
 
-const common = {
+const config: Config.InitialOptions = {
   testEnvironment: "jsdom",
-  transform:       {
+  collectCoverage: true,
+  transform: {
     "^.+\\.(t)sx?$": "@swc/jest",
   },
   setupFilesAfterEnv: ["<rootDir>src/test_setup.ts"],
-  testMatch:          ["<rootDir>/src/**/*.(spec|test).ts?(x)"],
-}
-
-const config: Config.InitialOptions = {
```

Comment:
This could have been removed when we removed `useCourseSearch`. That hook had some logic to support two different versions of the `history` package.